### PR TITLE
Add sample docstrings to eg samples

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_domain_topic_sample.py [hunk 1]
```diff
@@ -1,40 +1,54 @@
...
+                    )
+            event_list.append(event)
+
+        # publish list of events
+        client.send(event_list)
+        print("Batch of size {} published".format(len(event_list)))
+        time.sleep(randint(1, 5))
+
+if __name__ == '__main__':
+    publish_event()
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_domain_topic_sample.py:54]:
```suggestion
    publish_event()

```

File: sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_custom_schema_events_to_topic_sample.py [hunk 2]
```diff
@@ -1,44 +1,55 @@
...
+            event_list.append(event)
+
+        # publish list of events
+        client.send(event_list)
+        print("Batch of size {} published".format(len(event_list)))
+        time.sleep(randint(1, 5))
+
+
+if __name__ == '__main__':
+    publish_event()
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_custom_schema_events_to_topic_sample.py:55]:
```suggestion
    publish_event()

```

File: sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_event_grid_events_to_custom_topic_sample.py [hunk 3]
```diff
@@ -19,22 +26,26 @@
...
+                    )
+            event_list.append(event)
+
+        # publish list of events
+        client.send(event_list)
+        print("Batch of size {} published".format(len(event_list)))
+        time.sleep(randint(1, 5))
+
+if __name__ == '__main__':
+    publish_event()
```
Comment (rakshith91) [inline sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_event_grid_events_to_custom_topic_sample.py:51]:
```suggestion
    publish_event()

```

File: sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1b_publish_custom_events_to_a_topic_with_signature.py [hunk 4]
```diff
@@ -1,8 +1,20 @@
+"""
+FILE: cs1b_publish_custom_events_to_a_topic_with_signature.py
+DESCRIPTION:
+    These samples demonstrate sending an EventGrid Event using a shared access signature for authentication.
+USAGE:
+    python cs1b_publish_custom_events_to_a_topic_with_signature.py
+    Set the environment variables with your own values before running the sample:
+    1) EG_ACCESS_KEY - The access key of your eventgrid account.
```
Comment (KieranBrantnerMagee) [inline sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1b_publish_custom_events_to_a_topic_with_signature.py:8]: we specify these for some but not for others?

Comment (rakshith91) [reply sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1b_publish_custom_events_to_a_topic_with_signature.py:8]: yeah - consumer samples don't have any env variables - these are file specific

File: sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1_publish_custom_events_to_a_topic.py [hunk 5]
```diff
@@ -1,8 +1,20 @@
+"""
+FILE: cs1_publish_custom_events_to_a_topic.py
```
Comment (KieranBrantnerMagee) [inline sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1_publish_custom_events_to_a_topic.py:2]: Oh!  one thing I _totally_ forgot.  Copyright notice.

Comment (rakshith91) [reply sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1_publish_custom_events_to_a_topic.py:2]: adding

**Review by KieranBrantnerMagee — APPROVED** [review]

**Pull request merged**